src/test/java/uk/ac/ebi/subs/ena/EnaAgentAssayDataSubmissionsProcessorTest.java

```diff
@@ -90,6 +91,7 @@ public void whenUploadedFilesHasData_ThenItWillBeInsertedIntoFileMetadata() thro
         study.setId(UUID.randomUUID().toString());
         uk.ac.ebi.subs.data.Submission submission = new uk.ac.ebi.subs.data.Submission();
         submission.setTeam(team);
+        submission.setSubmissionDate(new Date());
```

Comment:
Is it mandatory to set the `submissionDate` on a submission? What happens if it is null?

Reply:
at first it was mandatory. thats why i updated the code in this project. but later when i looked the at the ena-validator project i realized this should not be a compulsory. i forgot to revert the code in this project but then i thought its probably not bad thing to keep it around. it kind of makes sense to set a submission date on a submission :)

let me know if you think its not necessary.

Reply:
Ok, it is fine. You can leave it there.